Source/PINDiskCache.m

```diff
@@ -1069,8 +1069,6 @@ - (void)setObject:(id)object forKeyedSubscript:(NSString *)key
 
 - (void)setObject:(id <NSCoding>)object forKey:(NSString *)key withAgeLimit:(NSTimeInterval)ageLimit fileURL:(NSURL **)outFileURL
 {
-    NSAssert(ageLimit <= 0.0 || (ageLimit > 0.0 && _ttlCache), @"ttlCache must be set to YES if setting an object-level age limit.");
```

Comment:
I had to remove this assertion because `_ttlCache` is set asynchronously. If you tried to call -setObject immediately after instantiating the cache, the assertion would throw.